src/MessagingAvatar/utils.ts

```diff
@@ -0,0 +1,70 @@
+/** A utility function that can be used with MessagingAvatar to convert a name into initials. */
+export function convertNameToInitials(name: string): string {
```

Comment:
Given how useful this is and that we're going to need it in FamPo, I figured it made sense to add it as a util we export from Dewey.

Reply:
I can't wait to see the revisions this one undergoes. Turns out that 'convert an unrestricted name input to initials' is quite thorny. 😄